sbnana/CAFAna/Core/OscCalcSterileApprox.h

```diff
@@ -75,7 +87,21 @@ namespace ana
     TMD5* GetParamsHash() const override {return calc.GetParamsHash();}
   };
 
-  OscCalcSterileApproxAdjustable* DefaultSterileApproxCalc();
+  inline SterileOscAngles operator|(const SterileOscAngles a, const SterileOscAngles b)
+  {
+    int a_int = static_cast<int>(a);
+    int b_int = static_cast<int>(b);
+    return static_cast<SterileOscAngles>(a_int | b_int);
+  }
+
+  inline SterileOscAngles operator&(const SterileOscAngles a, const SterileOscAngles b)
+  {
+    int a_int = static_cast<int>(a);
+    int b_int = static_cast<int>(b);
+    return static_cast<SterileOscAngles>(a_int & b_int);
+  }
```

Comment:
Does it ever make sense to `&` two angle configs?

Reply:
I use it to check which angles are set 
```
bool thetaEE_set = (angle & SterileOscAngles::kSinSq2ThetaEE) == SterilOscAngles::kNone;
```